open-health-manager-app/src/main/java/org/mitre/healthmanager/service/FHIRPatientConsentService.java

```diff
@@ -0,0 +1,293 @@
+    public Page<FHIRPatientConsentDTO> findAll(Pageable pageable) {
+    	List<FHIRPatientConsentDTO> results = findAll();
+        
+        List<FHIRPatientConsentDTO> list = results
+        	.stream()
+            .skip(pageable.getPageSize() * pageable.getPageNumber())
+       	    .limit(pageable.getPageSize())
+       	    .collect(Collectors.toList());
+
+        return new PageImpl<>(list, pageable, results.size());
+    }
+    
+    /**
+     * Get all the fHIRPatientConsents.
+     *
+     * @return the list of entities.
+     */
+    public List<FHIRPatientConsentDTO> findAll() {
+        log.debug("Request to get all FHIRPatientConsents");
+        IFhirResourceDao<Consent> resourceDAO = myDaoRegistry.getResourceDao(Consent.class);
+        RequestDetails requestDetails = SystemRequestDetails.forAllPartition();
+        SearchParameterMap searchParameterMap = new SearchParameterMap();
+        // reference:identifier search does not seem to work, for now manually filter
+        //SearchParameterMap searchParameterMap = new SearchParameterMap("organization", new ReferenceParam("identifier", "urn:mitre:healthmanager|"));
+        		//.add(Constants.PARAM_OFFSET, new NumberParam(pageable.getPageNumber())) not available
+        		//.add(Constants.PARAM_COUNT, new NumberParam(pageable.getPageSize()));
+        IBundleProvider results = resourceDAO.search(searchParameterMap, requestDetails);
+        
+        List<FHIRPatientConsentDTO> list = results.getAllResources()
+        		.stream()
+        		.map(resource -> (Consent) resource)
+        		.filter(consent -> consent.getOrganizationFirstRep() != null)
+        		.filter(consent -> consent.getOrganizationFirstRep().getIdentifier() != null)
+        		.filter(consent -> consent.getOrganizationFirstRep().getIdentifier().getSystem() != null)
+        		.filter(consent -> consent.getOrganizationFirstRep().getIdentifier().getSystem().equals("urn:mitre:healthmanager"))
+        	    .map(consent -> {
+        	    	return fhirPatientConsentMapper.toDto(consent);          	    	
+        	    })
+        	    .collect(Collectors.toList());
+
+        return list;
+    }
+
+    /**
+     * Get one fHIRPatientConsent by id.
+     *
+     * @param id the id of the entity.
+     * @return the entity.
+     */
+    public Optional<FHIRPatientConsentDTO> findOne(String id) {
+        log.debug("Request to get FHIRPatientConsent : {}", id);
+    	Consent consent = findById(id);    
+    	FHIRPatientConsentDTO consentDTO = fhirPatientConsentMapper.toDtoEagerLoad(consent);      	
+    	return Optional.of(consentDTO);
+    }
+
+    /**
+     * Delete the fHIRPatientConsent resource by id.
+     *
+     * @param id the id of the resource.
+     */
+    public void delete(String id) {
+        log.debug("Request to delete FHIRPatientConsent resource : {}", id);
+    	IFhirResourceDao<Consent> resourceDAO = myDaoRegistry.getResourceDao(Consent.class);
+    	resourceDAO.delete(new IdType(id));
```

Comment:
Should this also be deleting the patient consent entry in the jhipster database? It looks like this is only deleting it off the fhir server